resources/views/components/tables/rows/desktop/address.blade.php

```diff
@@ -1,5 +1,5 @@
-<x-general.identity :model="$model" :without-truncate="$withoutTruncate ?? false">
-    @if ($model->username())
+<x-general.identity :model="$model" :without-truncate="$withoutTruncate ?? false" :without-username="$withoutUsername ?? false">
+    @if ($model->username() && !($withoutUsername ?? false))
```

Comment:
As before, can we add props?

```suggestion
@props([
    'model',
    'withoutUsername' => false,
    'withoutTruncate' => false,
])

<x-general.identity :model="$model" :without-truncate="$withoutTruncate" :without-username="$withoutUsername">
    @if ($model->username() && !$withoutUsername)
```